docs/api/paths/admin/db-_user-name-all_channels.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022-Present Couchbase, Inc.
+#
+# Use of this software is governed by the Business Source License included
+# in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+# in that file, in accordance with the Business Source License, use of this
+# software will be governed by the Apache License, Version 2.0, included in
+# the file licenses/APL2.txt.
+parameters:
+  - $ref: ../../components/parameters.yaml#/db
+  - $ref: ../../components/parameters.yaml#/user-name
+get:
+  summary: Get all channels for a user
+  description: |-
+    Retrieve all channels that a user has access to.
+
+    Required Sync Gateway RBAC roles:
+
+    * Sync Gateway Architect
+    * Sync Gateway Application
+    * Sync Gateway Application Read Only
+  responses:
+    '200':
+      $ref: ../../components/responses.yaml#/User
```

Comment:
This doesn't match the format of the API repsonse. We'll have to add another response type for this API.

Reply:
I initially did the docs for the admin api, but I got comments on the framework PR #6639  saying since the API is unsupported then maybe we shouldnt have docs for it. Should I re-add diagnostic api docs?